src/main/java/com/seller/panel/dto/SignUpRequest.java

```diff
@@ -2,16 +2,23 @@
 
 import lombok.Data;
 
+import javax.validation.constraints.NotBlank;
+
 @Data
 public class SignUpRequest {
 
+    @NotBlank(message = "Must not be empty")
```

Comment:
Could have made one string constant 😃

Reply:
Yes I am doing the next PR, in which I am changing/adding endpoints

Reply:
In which also I am trying to read the messages from properties file